Makefile

```diff
@@ -89,10 +94,10 @@ deploy: # Deploy/redeploy previously built REANA cluster.
 	docker images | grep '<none>' | awk '{print $$3;}' | xargs -r docker rmi && \
 	reana-cluster -f ${PWD}/../reana-cluster/reana_cluster/configurations/reana-cluster-latest.yaml init --traefik
 
-example: # Run a simple "hello world" analysis example.
+example: # Run a REANA example. By default all REANA examples are executed.
 	source ${HOME}/.virtualenvs/${VENV_NAME}/bin/activate && \
 	eval $$(reana-dev setup-environment) && \
-	reana-dev run-example -c r-d-helloworld -s ${TIME_SLEEP}
+	echo "reana-dev run-example -c $${DEMO:-DEMO} -s ${TIME_SLEEP}"
```

Comment:
One last thing, but you need to rebase anyway: :wink: 

You could use:

```
DEMO ?=  DEMO
```

in the beginning of the script, like the other options (e.g. `VENV_NAME` and friends), for consistency... In this way all the input parameters are localised in one part of the script